health.go

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package mainflux
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+const (
+	contentType     = "Content-Type"
+	contentTypeJSON = "application/json"
```

Comment:
Content type can be 
application/health+json
 https://datatracker.ietf.org/doc/html/draft-inadarei-api-health-check-05#section-3